src/lib/backward-search/backward-search.spec.ts

```diff
@@ -371,7 +374,9 @@ describe("Backward Search Tests", () => {
         }
 
         for (let index = 1; index < 100; index++) {
-            largeLearningUnits[index].requiredSkills.push(largeSkillMap[index - 1]);
+            largeLearningUnits[index].requiredSkills = new And([
```

Comment:
Why And(variable) if only one variable is stored in And? Can we omit the And?

Reply:
Yes, I removed the And.